testing/base/source_gen.cpp

```diff
@@ -161,24 +160,25 @@ auto SourceGen::ClassGenState::GetValidTypeName() -> llvm::StringRef {
 }
 
 // We want to synthesize shuffled type names from the class names that will be
```

Comment:
This also generates the shuffled class names themselves. Perhaps include that too? Maybe also make it clear in this intro that this other list of shuffled type names will be used for type references?

Reply:
Yeah, tried a better initial sentence.